torch/quantization/quantize.py

```diff
@@ -66,7 +66,7 @@ def propagate_qconfig_(module, qconfig_dict=None):
     """
     if qconfig_dict is None:
         qconfig_dict = {}
-    _propagate_qconfig_helper(module, qconfig_dict)
+    _propagate_qconfig_helper(module, qconfig_dict, white_list)
```

Comment:
Can you remove the if check on qconfig_dict and set the default to {}?

Reply:
Seems set default to {} will result in lint error:
`[B006] Do not use mutable data structures for argument defaults. They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.`